lan867x/examples/lan867x_tcp_sockets/server/main/server.c

```diff
@@ -69,26 +76,52 @@ void app_main(void)
     // Stop dhcp client and set a static IP address
     esp_netif_dhcpc_stop(eth_netif);
     esp_netif_set_ip_info(eth_netif, &ip_info);
+#if CONFIG_EXAMPLE_LAN867X_USE_PLCA
+    // Configure PLCA as coordinator
+    uint8_t plca_nodes_count = CONFIG_EXAMPLE_LAN867X_PLCA_NODE_COUNT;
+    esp_eth_ioctl(eth_handles[0], LAN867X_ETH_CMD_S_PLCA_NCNT, &plca_nodes_count);
+    uint8_t plca_id = 0;
+    esp_eth_ioctl(eth_handles[0], LAN867X_ETH_CMD_S_PLCA_ID, &plca_id);
+    uint8_t plca_max_burst_count = 0;
+    esp_eth_ioctl(eth_handles[0], LAN867X_ETH_CMD_S_MAX_BURST_COUNT, &plca_max_burst_count);
+    bool plca_en = false;
```

Comment:
Should be enabled. I have a feeling, I've already commented it.

Reply:
Updated.